src/main/scala/org/broadinstitute/clio/model/authorization.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.clio.model
+
+import akka.http.scaladsl.model.headers.{ModeledCustomHeader, ModeledCustomHeaderCompanion}
+
+import scala.util.Try
+
+
+trait OidcHeader {
```

Comment:
It used to have more authy named things in it.
So I should rename it 'oidcHeaders.scala'?